tests/unit_test.py

```diff
@@ -213,6 +213,7 @@ def test_config_creation():
         gpu=7,
         instascale=True,
         machine_types=["cpu.small", "gpu.large"],
+        image_pull_secrets={"unit-test-pull-secrets"},
```

Comment:
Should that better be `["unit-test-pull-secret"]`?

Reply:
Thanks @astefanutti 
I've made the adjustment